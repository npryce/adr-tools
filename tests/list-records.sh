adr list
adr new first
adr list
adr new second
adr list
adr new third
adr list
adr list -r