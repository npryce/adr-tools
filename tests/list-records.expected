adr list
The doc/adr directory does not exist
adr new first
doc/adr/0001-first.md
adr list
doc/adr/0001-first.md
adr new second
doc/adr/0002-second.md
adr list
doc/adr/0001-first.md
doc/adr/0002-second.md
adr new third
doc/adr/0003-third.md
adr list
doc/adr/0001-first.md
doc/adr/0002-second.md
doc/adr/0003-third.md
adr list -r
doc/adr/0003-third.md
doc/adr/0002-second.md
doc/adr/0001-first.md
